components/k8s-dashboard-proxy/cmd/reverseproxy/reverseproxy.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+)
+
+const (
+	dashboardURL   = "http://localhost:30000"
+	secretFilePath = "/var/run/secrets/kubernetes.io/serviceaccount/token"
```

Comment:
Please make it configurable (as a parameter) with this value as a default value.

Reply:
@radufa Actually, I've been thinking about, but for me, after all, it's a single (or 3) configuration parameter, doesn't deserve the hassle of being moved to a config file and introducing a new dependency to read the config. especially if the changes to the config value will require recompilation anyways, and, one of the Golang `const` uses is to hold config values.

Reply:
@radufa Do you want it to be a `cmd` passed variable, so it can be changed from the Helm chart?

Reply:
It's fine as a "cmd" passed variable, cause it could be changed in the future.